azurerm_express_route_circuit_peering -  support for bandwidth_in_gbps, express_route_port_id (#12289)
terraform-providers/terraform-provider-azurerm: website/docs/r/express_route_circuit.html.markdown

* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.

* `sku` - (Required) A `sku` block for the ExpressRoute circuit as documented below.

> * `service_provider_name` - (Optional) The name of the ExpressRoute Service Provider. Any change to the configuration will recreate the resource.

Reviewer: ```suggestion
* `service_provider_name` - (Optional) The name of the ExpressRoute Service Provider. Changing this forces a new resource to be created.
```

> * `peering_location` - (Optional) The name of the peering location and **not** the Azure resource location. Any change to the configuration will recreate the resource.

Reviewer: ```suggestion
* `peering_location` - (Optional) The name of the peering location and **not** the Azure resource location. Changing this forces a new resource to be created.
```

* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.

* `sku` - (Required) A `sku` block for the ExpressRoute circuit as documented below.

* `service_provider_name` - (Optional) The name of the ExpressRoute Service Provider.

* `peering_location` - (Optional) The name of the peering location and **not** the Azure resource location.

* `bandwidth_in_mbps` - (Optional) The bandwidth in Mbps of the circuit being created on the Service Provider.

~> **NOTE:** Once you increase your bandwidth, you will not be able to decrease it to its previous value.

~> **NOTE:** The `service_provider_name`, the `peering_location` and the `bandwidth_in_mbps` should be set together and they conflict with `express_route_port_id` and `bandwidth_in_gbps`.

> * `allow_classic_operations` - (Optional) Allow the circuit to interact with classic (RDFE) resources.

Reviewer: ```suggestion
* `allow_classic_operations` - (Optional) Allow the circuit to interact with classic (RDFE) resources. Defaults to `false`.
```

> ~> **NOTE:** The `allow_classic_operations` cannot be set while using the Express Route Port.

Reviewer: Can we add this constraint back in the schema definition?

> * `express_route_port_id` - (Optional) The ID of the Express Route Port which this circuit should be based on.

Reviewer: ```suggestion
* `express_route_port_id` - (Optional) The ID of the Express Route Port which this Express Route Circuit is based on.
```
